atlp-rwanda/destructors-ec-bn
# #184637120 postgresDB and sequelize ORM configuration

#### What does this PR do?

- Configuration of postgresDB and sequelize ORM

#### Description of Task to be completed?

- Configure The App to use PostgreSQL as DB Engine and Sequelize ORM

#### How should this be manually tested?

- clone  the repository 
- checkout to develop branch
- pull from the develop branch
- checkout to the new branch you want to work in 
- to install the dependencies used run in your terminal ` npm install `
- to run app run in your terminal ` npm run dev `


#### To test the Database

- first create .env file
- check .env.example how to add your database url
- In you terminal  navigate to database folder by running `cd src/database`
- In database folder you can run  `sequelize db:migrate to create table`

NOTE: in database folder is where to create tables

#### What are the relevant pivotal tracker strory?

#184637120
